cos-event/run

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Env Vars:
+# REPOSITORY: name of the image registry/namespace to get the images
+# COS_ID: If set, specifies the full CRN of a Cloud Object Storage instance to use
+
+function project_name() {
+  ibmcloud ce project current | awk -F: '/Project Name/{ print $2 }'
+}
+
+function project_id() {
+  set +x
+  ibmcloud ce project get --name $(project_name)| awk '/^ID/{ print $2 }'
+  set -x
+}
+
+# Clean up previous run
+function clean() {
+  set +x
+  DCOS=$1
+  POLICY=$2
+  echo "Cleaning...be patient"
+
+  # Delete the event subscription
+  ibmcloud ce sub cos delete -n cos-sub -f --wait=true > /dev/null 2>&1 || true
+
+  # Delete the IAM auth policy
+  if [[ $POLICY != "" ]]; then
+    ibmcloud iam authorization-policy-delete $POLICY --force > /dev/null 2>&1 || true
+  fi
+
+  # Delete the app
+  ibmcloud ce app delete --name cos-app --force > /dev/null 2>&1 || true
+
+  # Delete the COS bucket
+  ibmcloud cos bucket-delete --bucket $(project_id) --force > /dev/null 2>&1 || true
+
+  # Delete the COS instance
+  if [[ $DCOS -gt 0 ]]; then
+    ibmcloud resource service-instance-delete codeengine-cos -f -q > /dev/null 2>&1 || true
+  fi
+
+  rm -f out
+}
+
+clean 0
+[[ "$1" == "clean" ]] && exit 0
+
+set -ex
+export REPOSITORY=${REPOSITORY:-ibmcom}
+
```

Comment:
minor thing but if you add:
```
export PROJECT_NAME=$(ibmcloud ce project current | awk -F: '/Project Name/{ print $2 }')
```
here then you can remove the function above and you'll only end up calling it once instead of multiple time.
Same for project_id

Reply:
Lemme rework the run script later as a separate PR.  I think there are a number of simplification paths we could pursue...